servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -502,6 +508,33 @@ public long createFile(TachyonURI path, long blockSizeBytes, boolean recursive,
   }
 
   /**
+   * Reinitializes the blocks of an existing open file.
```

Comment:
What calls this? What happens if the file is already complete?

What does "reinitialize" blocks mean?